inspirehep/config.py

```diff
@@ -69,11 +69,11 @@ def _(x):
 
 # Celery
 # ======
-BROKER_URL = "amqp://guest:guest@localhost:5672//"
-CELERY_RESULT_BACKEND = "amqp://guest:guest@localhost:5672//"
+CELERY_BROKER_URL = "pyamqp://guest:guest@localhost:5672//"
+CELERY_RESULT_BACKEND = "rpc://guest:guest@localhost:5672//"
 CELERY_ACCEPT_CONTENT = ['json', 'msgpack', 'yaml']
```

Comment:
Do we really need this? I don't see why we can't just use one type of messages (of the three, `msgpack` should be the one that guarantees the smallest messages).